### PR TITLE
Improve explanation of how this document relates to laws and regulation.

### LINKED ISSUE #168: How This Document Fits In
> Privacy is covered by legal frameworks and this document recognises that existing data protection laws take precedence for legal matters. However, because the Web is global, we benefit from having shared concepts to guide its evolution as a system built for the people using it ([[RFC8890](https://w3ctag.github.io/privacy-principles/#bib-rfc8890)]). A clear and well-defined view of privacy on the Web, informed by research, can hopefully help all the Web's participants in different legal regimes. Our shared understanding is that the law is a floor, not a ceiling.

I'd say this in a slightly different way. Something like:

> Privacy on the Web is primarily regulated by two forces: the architectural capabilities that the Web platform exposes (or does not), and laws in the various jurisdictions where it is used. These regulatory mechanisms are separate; a law in one country does not (and should not) change the whole Web's architecture, and likewise Web specifications cannot override any given law (although their implementation can make some laws difficult to enforce). The Web is not merely an implementation of a legal privacy regime; it has distinct features and guarantees driven by shared values that often exceed legal requirements.
>
> However, the overall goal of privacy on the Web is served best when technology and law compliment each other. This document seeks to establish shared concepts as an aid to technical efforts to regulate privacy on the web, and may also be useful in pursuing alignment with an between legal regulatory regimes.

### PR DESCRIPTION
**Pull request by jyasskin**
Fixes #168. I made two minor tweaks to @mnot's suggested wording.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/210.html" title="Last updated on Jan 18, 2023, 8:11 PM UTC (87319f6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/210/0c8f4e5...jyasskin:87319f6.html" title="Last updated on Jan 18, 2023, 8:11 PM UTC (87319f6)">Diff</a>

### CODE REVIEW

**Comment by mnot** [comment]
LGTM, thx.

**Comment by wseltzer** [comment]
Lessig notwithstanding, I wonder if there's a word other than "regulate" to reflect the technical/architectural framework-setting. "Regulate" still connotes governmental action, to my US ears. s/regulated/affected/ ?

**Comment by jyasskin** [comment]
We'll add a citation to Code and Other Laws of Cyberspace, and merge this.

**Review by rhiaro — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -460,12 +460,18 @@
 not be taken as an indication that privacy is always more important than other ethical web principles, and
 this document doesn't address how to balance the different ethical web principles if they come into conflict.
 
-Privacy is covered by legal frameworks and this document recognises that existing data
-protection laws take precedence on legal matters. However, because the Web is global, we
-benefit from having shared concepts to guide its evolution as a system built for the people who use it
-([[?RFC8890]]). A clear and well-defined view of privacy on the Web, informed by research,
-can hopefully help all the Web's participants in different legal regimes. Our shared
-understanding is that the law is a floor, not a ceiling.
+Privacy on the Web is primarily regulated by two forces: the architectural capabilities that the Web
```
Comment (wseltzer) [inline index.html:463]: [[regulated]] cite to Code and Other Laws of Cyberspace, Lawrence Lessig (1999) https://lessig.org/product/code/

Comment (jyasskin) [reply index.html:471]: Done, thanks!

Comment (mnot) [reply index.html:471]: FWIW, he first outlined this AFAIK in 'The New Chicago School', and it's a much shorter read :) See https://www.journals.uchicago.edu/doi/pdf/10.1086/468039

**Pull request merged**